7_react/react1/week1/homework/todo-list/src/components/todolist.jsx

```diff
@@ -0,0 +1,16 @@
+// TodoList.jsx
```

Comment:
its great to have comments, but since we can read the name from the file this feels bit redundant